src/v2/BaseLoginRouter.js

```diff
@@ -32,11 +32,21 @@ import AppState from './models/AppState';
 import idx from 'idx';
 
 
-const introspectStateToken = (settings) => {
+const introspect = (settings) => {
+  const clientId = settings.get('clientId');
   const domain = settings.get('baseUrl');
+  const scopes = settings.get('scopes');
   const stateHandle = settings.get('stateToken');
+  const redirectUri = settings.get('redirectUri');
   const version = settings.get('apiVersion');
-  return idx.start({ domain, stateHandle, version });
+  
+  if (stateHandle) {
+    return idx.start({ domain, stateHandle, version });
+  } else if (settings.get('useInteractionCodeFlow')) {
+    return idx.start({ clientId, domain, scopes, stateHandle, redirectUri, version });
+  } else {
+    throw new Errors.ConfigError('Set "useInteractionCodeFlow" to true in configuration to enable the "interaction_code" flow.');
```

Comment:
need a better to prompt for this warning.. This should not be an error!!

Reply:
The error here is for devs only, I assume the above two cases are the only cases we should support.